src/components/Footer.js

```diff
@@ -25,29 +30,35 @@ const Footer = props => (
       </dl>
       <ul className="icons">
         <li>
-          <a href="https://twitter.com/mikelax" className="icon fa-twitter alt">
-            <span className="label">Twitter</span>
+          <a href="https://twitter.com/mikelax" className="bottom-none">
+          <IconContext.Provider value={{ className: "icon alt" }}>
```

Comment:
@Bobeta let's move this `IconContext.Provider` above the `<ul className="icons">` so this way we only have to use it once instead of around each icon usage. I think moving there so we don't need to repeat makes things DRYer while still having the usage fairly local.